yaml/roads.yaml

```diff
@@ -78,6 +78,14 @@ global:
       sidewalk: {col: sidewalk}
       sidewalk_left: {col: 'sidewalk:left'}
       sidewalk_right: {col: 'sidewalk:right'}
+      cutting:
+        case:
+          - when: {cutting: ['yes', 'right', 'left', 'both']}
```

Comment:
Can this work more like bike lanes – where it's yes, right, left and if both then it'd just be yes? (Please verify, this is from memory.)

Reply:
I was just setting `cutting` to `true` if it matched any of those (or omitting if it didn't match). I've updated in 8c79ac2 to make the values `yes`, `left` or `right`.

Looking at the properties for `cycleway`, it looks like we export a `cycleway_both` property, so we don't collapse that down onto `cycleway`. Perhaps that's something we should come back to on a ☔ (although now I suppose it'd be a breaking change, we'd need to wait for v2.0).